### ISSUE TITLE
Add an option to quicky save the file in discard changes dialog

### ISSUE THREAD

**Issue by MarcAnt01** [ikarago/Notepad#27]

**Comment by prayaas-a** [comment]
Good idea. I think the behavior should align more closely with the Win32 Notepad as that's the expected behavior for most users of this app. A prompt that has the options "Save", "Don't Save" or "Discard", as well as "Cancel."
It's better to have a nondestructive action as the first ("Yes") option which is the opposite of how it is currently.
